Automattic/wp-calypso
# Remove/publicize gutenblock effects middleware

#### Changes proposed in this Pull Request

I think we don't need the effect and associated middlwware for `refreshConnections`. Instead, we can make that action a generator, with the same implementation as the `getConnections` resolver (which we can then simply alias).

#### Testing instructions

* Check out this branch.
* Spin up local Jetpack with the `add/publicize-rest-api-3` branch.
* Make sure Jetpack is active and connected.
* Run `npm run sdk -- gutenberg client/gutenberg/extensions/presets/jetpack/ -- --output-dir=DIR_TO_JETPACK/jetpack/_inc/blocks` where `DIR_TO_JETPACK` is your Jetpack directory.
* Start writing a new post.
* In Gutenberg, click on 'Publish...'
* Open the browser console's network tab, and clear it
* Click on 'Connect new service', which will open a new tab with Publicize connections.
* Close that new tab, landing back in the tab with your new post. In the browser console's network tab, verify that after closing the new tab, the `wp/v2/posts/{postId}/?context=edit` endpoint was hit.
* Verify that the rest of the Publicize extension also still works as before.
